client/src/components/new-request/search.vue

```diff
@@ -1,89 +1,163 @@
+                        :hide-details="!errors.includes('entity_type_cd')"
+                        :items="entityConversionTypeOptions"
+                        :placeholder="isConversion ? 'Please choose a conversion type' : ''"
+                        @change="clearErrors()"
+                        filled
+                        v-model="entity_type_cd">
+                <template slot="item" slot-scope="data">
+                  <v-tooltip
+                          :right="isScreenLg"
+                          :left="!isScreenLg"
+                          :disabled="!data.item.blurbs"
+                          :content-class="!isScreenLg ? 'left-tooltip' : ''">
+                    <template v-slot:activator="scope">
+                      <span v-on="scope.on"
+                            class="list-item"
+                            :class="{ 'entity-type-info': data.item.value === 'INFO' }">
+                        {{ data.item.text }}
+                      </span>
+                    </template>
+                    <div v-for="(item, index) in (location === 'IN' ? data.item.intBlurb : data.item.blurbs)"
+                         :key="`Blurb-${index}`">
+                      <span :class="{ 'tooltip-bullet': index !== 0}">
+                        {{ item }}
+                      </span>
+                    </div>
+                  </v-tooltip>
+                </template>
+              </v-select>
+            </div>
+          </template>
+          <span>{{ entityConversionText }}</span>
+        </v-tooltip>
       </v-col>
       <NameInput :class="inputCompClass"
                  id="name-input-component"
                  class="mb-n7"/>
     </v-row>
-    <v-row no-gutters class="ma-0 pa-0 mt-n3" align="center">
+    <v-row no-gutters>
       <v-col cols="4">
-      <v-tooltip bottom open-delay="300">
-        <template v-slot:activator="{on}">
-            <v-checkbox v-model="isPersonsName"
-                        v-on="on"
-                        id="name-checkbox"
-                        class="copy-small px-0 mx-0"
-                        label="The name is a person's name" />
-        </template>
-        <p class="py-0 my-0">Check this box if you are...</p>
+        <v-tooltip bottom content-class="bottom-tooltip mt-n5">
+          <template v-slot:activator="{ on }">
+            <v-checkbox
+                    v-model="isPersonsName"
+                    id="name-checkbox"
+                    class="copy-small mr-5"
+                    v-slot:label v-on="on">
+              <template>
+                <span v-on="on">Name is a person's name</span>
+              </template>
+            </v-checkbox>
+          </template>
+          <p>Check this box if you are...</p>
           <ul>
             <li>Incorporating under your own name (eg. DR. JOE SMITH INC.)</li>
             <li>The name contains one or more names. (eg. BLAKE, CHAN & DOUGLAS INC.)</li>
```

Comment:
Generally `&amp;` should be used instead of `&` but if works, leave it.

Reply:
It does seem to work. I wonder why, if it's not correct tho. Will look into it!

Reply:
https://mrcoles.com/blog/how-use-amersands-html-encode/
https://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp

Reply:
"should" not "must" FWIW

Reply:
Updated!